kotlinpoet-classinspector-reflective/src/main/kotlin/com/squareup/kotlinpoet/classinspector/reflective/ReflectiveClassInspector.kt

```diff
@@ -191,24 +192,32 @@ class ReflectiveClassInspector private constructor() : ClassInspector {
     return exceptionTypes.orEmpty().mapTo(mutableListOf()) { it.asTypeName() }
   }
 
-  override fun enumEntry(enumClassName: ClassName, memberName: String): ImmutableKmClass? {
+  override fun enumEntry(enumClassName: ClassName, memberName: String): EnumEntryData {
```

Comment:
This change is of course breaking but seeing as it's annotated with `@KotlinPoetMetadataPreview` it feels ok.